src/reducers/index.js

```diff
@@ -43,4 +44,11 @@ const rootReducers = combineReducers({
   errorNotification: errorNotificationReducer,
 });
 
-export default rootReducers;
+const rootReducer = (state, action) => {
+  if (action.type === RESET_WALLET)
```

Comment:
This lets us reset the app without refreshing the page. This is currently only used when the wallet config is cleared from the state and the sessionStorage. setting the state to undefined will reset to the default, i.e. the initialState, set in each reducer.